Automattic/wp-calypso
# My Home: Remove old locations and cards

#### Changes proposed in this Pull Request

Removes the unused cards and locations after activating the new My Home layout. It also removes the experimental flag.

#### Testing instructions

- Go to My Home for a variety of sites (unlaunched, launched, established, WP for Teams).
- Make sure there are no changes when compared to production.

Fixes #41776
